packages/css-doc-page/src/assets/markdown/accordion/accordionMark.md

```diff
@@ -0,0 +1,24 @@
+```interactive
+<div className="wrapper">
+          <div className="accordion">
+            <Disclosure
+              title="Label 1"
+              description="Lorem Ipsum is simply dummy text of the printing  "
+            ></Disclosure>
+            <Disclosure
+              title="Label 2"
+              description="Lorem Ipsum is simply dummy text of the printing  "
+            ></Disclosure>
+            <Disclosure
+              title="Label 3"
+              description="Lorem Ipsum is simply dummy text of the printing  "
+            ></Disclosure>
+            <Disclosure
+              title="Label 4"
+              description="Lorem Ipsum is simply dummy text of the printing  "
+            ></Disclosure>
+          </div>
+        </div>
+
+
+```
```

Comment:
The use of markdowns is a convention that we have to consolidate. Currently we have several ways to consume the documentation. Honestly, I consider that if we put all the content in the `.md` file much better.